test/input/sf-light/sf-light.conf

```diff
@@ -33,7 +33,7 @@ beam.agentsim.agents.vehicles.bicycles.useBikes=false
 beam.agentsim.agents.vehicles.beamFuelTypesFile = ${beam.inputDirectory}"/beamFuelTypes.csv"
 beam.agentsim.agents.vehicles.beamVehicleTypesFile = ${beam.inputDirectory}"/sample/5k/vehicleTypes.csv"
 beam.agentsim.agents.vehicles.beamVehiclesFile = ${beam.inputDirectory}"/sample/1k/vehicles.csv.gz"
-beam.agentsim.agents.population.beamPopulationFile = ${beam.inputDirectory}"/population-100.csv"
+//beam.agentsim.agents.population.beamPopulationFile = ${beam.inputDirectory}"/population-100.csv"
```

Comment:
Commenting conf files is done via # not // - at least afaik

Reply:
fixed